Cargo.toml

```diff
@@ -23,6 +23,7 @@ merlin = "3.0.0"
 
 thiserror = "1.0"
 displaydoc = "0.2"
+byteorder = "1.4.3"
```

Comment:
What do you need this for? I seem to be able to use the `to_le_bytes()` without this import